### PR TITLE
Dependency update 2022

### PR DESCRIPTION
**Pull request by chibenwa**

### CODE REVIEW

**Review by Arsnael — APPROVED** [review]

**Comment by chibenwa** [comment]
```
org.apache.james.backends.rabbitmq.ReactorRabbitMQChannelPoolTest.channelPoolShouldThrowAfterTimeoutWhenAllChannelsAreTaken

Expecting message to be:
  "Timeout waiting for idle object"
but was:
  "Timeout waiting for idle object, borrowMaxWaitDuration=PT5S"
  
org.apache.james.webadmin.dropwizard.MetricsRoutesTest.getShouldReturnSeveralMetric

java.lang.AssertionError: 

Expecting actual:
  ""
to contain:
  "# HELP hard Generated from Dropwizard metric import (metric=hard, type=com.codahale.metrics.Counter)
# TYPE hard gauge
hard 2.0
# HELP easy Generated from Dropwizard metric import (metric=easy, type=com.codahale.metrics.Counter)
# TYPE easy gauge
easy 1.0" 
	at org.apache.james.webadmin.dropwizard.MetricsRoutesTest.getShouldReturnSeveralMetric(MetricsRoutesTest.java:65)
```

**Comment by chibenwa** [comment]
The build was :green_apple: prior the rebase

File: server/blob/blob-s3/pom.xml [hunk 1]
```diff
@@ -33,8 +33,8 @@
     <name>Apache James :: Server :: Blob :: S3</name>
 
     <properties>
-        <s3-sdk.version>2.17.9</s3-sdk.version>
-        <netty.version>4.1.65.Final</netty.version>
+        <s3-sdk.version>2.17.107</s3-sdk.version>
+        <netty.version>4.1.72.Final</netty.version>
```
Comment (Arsnael) [inline server/blob/blob-s3/pom.xml:37]: The versions here don't match with the commit name

File: server/protocols/jmap-rfc-8621-integration-tests/jmap-rfc-8621-integration-tests-common/pom.xml [hunk 2]
```diff
@@ -65,7 +65,7 @@
         <dependency>
             <groupId>com.softwaremill.sttp.client3</groupId>
             <artifactId>okhttp-backend_${scala.base}</artifactId>
-            <version>3.0.0</version>
+            <version>3.3.18</version>
```
Comment (Arsnael) [inline server/protocols/jmap-rfc-8621-integration-tests/jmap-rfc-8621-integration-tests-common/pom.xml:68]: Suggestion commit name: `xmlunit 2.6.3 -> 2.8.4 & okhttp 3.0.0 -> 3.3.18`

Comment (chibenwa) [reply server/protocols/jmap-rfc-8621-integration-tests/jmap-rfc-8621-integration-tests-common/pom.xml:68]: okhttp is just a driver here, the core engine is sttp.

File: server/blob/blob-s3/pom.xml (hunk 1 above)
Comment (chibenwa) [reply server/blob/blob-s3/pom.xml:37]: Yes I had to allign those on reactor-netty version

**Comment by chibenwa** [comment]
Force pushed to reword commits following @Arsnael  remarks.

**Review by Arsnael — APPROVED** [review]

**Pull request merged**